### DISCUSSION TITLE
Emojik és a Windows - SK blog

### DISCUSSION THREAD

**Discussion by giscus** [skornel02/skornel02#8] [Comment section]
# Emojik és a Windows - SK blog

Minimális történelem, közepesen érdekes információk és a Windows emoji verziójának frissítése

https://skornel02.hu/posts/emojik-es-a-windows

**Comment by hgergely03** [comment]
*Elsőre* betölt! 🐱‍🐉

**Comment by hgergely03** [comment]
Ez itt az *első*!

**Reply by skornel02** [reply]
Hurray!!!
